metricbeat/tests/system/test_traefik.py

```diff
@@ -0,0 +1,36 @@
+import os
```

Comment:
We recently started to put this file directly into the module directory (see Elasticsearch) to have all pieces in one place. But still both works.